climada_petals/hazard/test/test_tc_tracks_forecast.py

```diff
@@ -69,10 +70,24 @@ def test_fetch_ecmwf(self):
         self.assertEqual(forecast.data[1].sid, '22S')
         self.assertEqual(forecast.data[1].name, 'HEROLD')
         np.testing.assert_array_equal(forecast.data[0].basin, 'S')
-        self.assertEqual(forecast.data[0].category, 'Tropical Depression')
-        self.assertEqual(forecast.data[0].run_datetime,
-                         np.datetime64('2020-03-19T12:00:00.000000'))
-        self.assertEqual(forecast.data[1].is_ensemble, True)
```

Comment:
why were these four lines deleted?

Reply:
I would expect you need to add a new test but nothing more

Reply:
You are absolutely right. this is a mistake.

Reply:
The lines have been reintroduced now.